File: solutions/c#/LRUCache/Node.cs
namespace LRUCacheNamespace
{
    public class Node<K, V>
    {
        public K Key { get; }
        public V Value { get; set; }
        public Node<K, V> Prev { get; set; }
        public Node<K, V> Next { get; set; }

        public Node(K key, V value)
        {
            Key = key;
            Value = value;
        }
    }
}

File: solutions/python/lrucache/node.py
from typing import TypeVar, Generic, Optional

K = TypeVar('K')
V = TypeVar('V')

class Node(Generic[K, V]):
    def __init__(self, key: K, value: V):
        self.key = key
        self.value = value
        self.prev: Optional['Node[K, V]'] = None
        self.next: Optional['Node[K, V]'] = None